spec/factories/edition_factory.rb

```diff
@@ -209,5 +209,9 @@
     trait :not_political do
       system_political { false }
     end
+
+    trait :pre_release do
+      document_type { build(:document_type, :pre_release, path_prefix: "/prefix") }
```

Comment:
Could you help me understand why we're setting a `path_prefix` here? It looks like it's redundant.

Reply:
It was, I'd seen it used in a lot of other places and assumed it was mandatory, gone now